protocol/messenger.go

```diff
@@ -4092,6 +4092,8 @@ func (m *Messenger) markAllRead(chatID string, clock uint64, shouldBeSynced bool
 	chat.UnviewedMessagesCount = 0
 	chat.UnviewedMentionsCount = 0
 
+	chat.LastMessage.Seen = true
```

Comment:
This fixes the MarkAllRead behaviour to make sure the LastMessage is marked as Seen too